report/renderer/dom.js

```diff
@@ -157,7 +157,7 @@ export class DOM {
       return;
     }
 
-    const allowedProtocols = ['https:', 'http:'];
+    const allowedProtocols = ['https:', 'http:', window.location.protocol];
```

Comment:
Is this mostly to account for `file:` protocols? Could we ignore `allowedProtocols` if the origin matches `window.location` instead?

Reply:
Yes, it was to account for `file:` when viewing a standalone report.

> Could we ignore allowedProtocols if the origin matches window.location instead?

Isn't this the same behavior?

Reply:
> Isn't this the same behavior?

I meant we could bypass `allowedProtocols` if the `url.origin` matches `location.origin`.